index.js

```diff
@@ -37,8 +37,9 @@ function corsGate(options) {
     const origin = (req.headers.origin || '').toLowerCase().trim();
 
     if (!origin) {
+      const allowSafe = typeof(options.allowSafe) === 'boolean' ? options.allowSafe : options.allowSafe(req, res);
```

Comment:
```suggestion
      const allowSafe = !!(typeof options.allowSafe === 'function' ? options.allowSafe(req) : options.allowSafe);
```

Reply:
Was the removal of `res` intentional in your suggestion? We actually use `res` in the one case where we use this function.

Reply:
rats; yeah, it was